server.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+const app = express();
+const port = 3000;
```

Comment:
:chefskiss: Glorious! This works as expected!

One thing to consider about the naming, and feel free to ping me about this because I can imagine that this might come across as confusing. This is a long comment so I'm going to give you the opportunity to read the unabridged or abridged version. Choose your own adventure!

### Unabridged
In ES5, we have two new variable types (in addition to `var`) which are `const` (constants, or immutable variables) and `let` (scoped variables but more importantly, mutable). Big difference between the two, you can redeclare the value of a variable declared with `let` but not `const`.

```
// This is okay.
let myFavoriteColor = 'green'
myFavoriteColor = 'blue'

// This is not.
const myFavoriteColor = 'green';
myFavoriteColor = 'blue'
```

Within constant variables though, we have times where the value of const _is_ variable (like saying `const myFavoriteColor = getMyFavoriteColor()`) and times where it's not (`const myFavoriteColor = 'green'`). Th difference here is that the first example that references `getMyFavoriteColor` is assigned a different value based on the return value of `getMyFavoriteColor`, which can be variable. It means that the value of `myFavoriteColor` _might_ be green, or might be red or blue. It all depends on the value returned from `getMyFavoriteColor`. And saying, "It all depends" is the same as saying "it's variable."

Now let's revisit `const myFavoriteColor = 'green'`. Something is different here. `myFavoriteColor` has an assigned value that isn't variable. It's completely static. These are called static constants, and they have a different naming convention. In many languages (Java, PHP, and yes, JavaScript) a common convention is to distinguish an immutable variable from an immutable constant by using upper snake case. Which means my code might look like this:

```
const MY_FAVORITE_COLOR = 'green'
const MY_FIRST_NAME = 'Jordan'
const currentDay = getCurrentDay()

const greeting = `Hello, my name is ${MY_FIRST_NAME}, today is ${currentDay}, and  my favorite color is ${MY_FAVORITE_COLOR}`
```

Note that there are four immutable variables here (that is, four variables have their value declared once and never change), but only two are statically constant. `MY_FAVORITE_COLOR` and `MY_FIRST_NAME` will never change, but the value of `currentDay` is variable based on the current day which means that, by definition, `greeting` is also variable.

None of this is required to move this forward, but that's just a convention to be aware of if you're interested.


### Abridged
Consider changing `port` to `PORT`.